docs/api/js/accounts-l1-l2.md

```diff
@@ -155,6 +155,24 @@ async getBaseCost(params: {
 | params.gasPrice (optional)  | The gas price of the L1 transaction that will send the request for an execute call.                    |
 | returns                     | The base cost in ETH for requesting the contract call.                                                 |
 
+## Claim Failed Deposit
+
+The `claimFailedDeposit` method withdraws funds from the initiated deposit, which failed when finalizing on L2.  
+If the deposit L2 transaction has failed it sends an L1 transaction calling claimFailedDeposit method of the L1 bridge, which results in returning L1 tokens back to the depositor, otherwise throws the error.
+
+### Input Parameters
+
+| Parameter |       Type          | Description                                                                               |
+| --------- |       ------------  | ----------------------------------------------------------------------------------------- |
+| depositHash       | `bytes32`   | 2 transaction hash of the failed deposit.                                    |
```

Comment:
I guess this line should be removed

Reply:
it is a doc for the SDK method, not contract one, so `depositHash` should be left and other params should be removed

Reply:
You are right